packages/react/src/table/docs/overview.mdx

```diff
@@ -922,13 +1110,25 @@ For more information, read the [Selectable tables with batch actions pattern](/p
 											</Checkbox>
 										</TableCell>
 										<TableCell as="th" scope="row" fontWeight="bold">
-											<TextLink href="#">REF-AB3CD4EF</TextLink>
+											<TextLink href="#" id={`REF-AB${idx}CD4EF`}>
```

Comment:
I'd love to rid public docs of the `idx` shortcut, if you have it in you.

Reply:
I actually think `i` would be better here, `index` best, mind you. `idx`, especially as part of an `id` value, still makes me think of "id-extension".